static/app/views/onboarding/setupDocs.tsx

```diff
@@ -263,10 +264,14 @@ function SetupDocs({search, route, router, location, ...props}: Props) {
     'onboarding-heartbeat-footer'
   );
 
-  const docsWithProductSelection = organization.features?.includes(
+  const docsWithProductSelection = !!organization.features?.includes(
     'onboarding-docs-with-product-selection'
   );
 
+  const loaderOnboarding = !!organization.features?.includes('onboarding-project-loader');
+
+  const jsDynamicLoader = !!organization.features?.includes('js-sdk-dynamic-loader');
```

Comment:
I think actually we could have used only one feature flag 🤔

Reply:
should I remove the other flag and do it based on this? But maybe it is still OK to leave this as I _think_ we'll flip the `js-sdk-dynamic-loader` one on by default soon!